app/javascript/packs/application.js

```diff
@@ -2,22 +2,39 @@
 
 import Vue from 'vue/dist/vue.esm';
 import VueI18n from 'vue-i18n';
-import Selector from '../selector.vue';
+import vSelect from 'vue-select';
 import Repository from '../repository.vue';
+import Dropdown from 'bp-vuejs-dropdown';
```

Comment:
Absolute imports should come before relative imports                  import/imports-first